examples/utils/generic.py

```diff
@@ -11,6 +11,44 @@
 
 # GENERIC UTILITIES
 
+def update_settings_with_users_authorization_info(ACCOUNT_ID='ACCOUNT_ID', 
+                                                  AUTH_TOKEN='AUTH_TOKEN', 
+                                                  MATERIALS_PROJECT_API_KEY='MATERIALS_PROJECT_API_KEY', 
+                                                  ORGANIZATION_ID='ORGANIZATION_ID'):
+    """
+    This function updates settings.py with the users authorization info.
+    It assumes 'settings.py' is located in the folder above this file's location.
+    Ex) '../settings.py'
+
+    Args:
+        ACCOUNT_ID (str): Users' ACCOUNT_ID
+        AUTH_TOKEN (str): Users' AUTH_TOKEN
+        MATERIALS_PROJECT_API_KEY (str): Users' MATERIALS_PROJECT_API_KEY
+        ORGANIZATION_ID (str): Users' ORGANIZATION_ID
+        settings_filepath (str): Path to settings.py
+
+    Returns:
+        None
+    """
+    
+    assert os.path.isfile('../settings.py')
+    
+    with open('../settings.py') as settings:
+        settings_filelines = settings.readlines()
+    with open('../settings.py', "w") as settings:
+        for line in settings_filelines:
+            if 'ACCOUNT_ID = ' in line:
+                settings.write('%s "%s"\n' % ("ACCOUNT_ID =", ACCOUNT_ID))
+            elif 'AUTH_TOKEN = ' in line:
+                settings.write('%s "%s"\n' % ("AUTH_TOKEN =", AUTH_TOKEN))
+            elif 'MATERIALS_PROJECT_API_KEY = ' in line:
+                settings.write('%s "%s"\n' % ("MATERIALS_PROJECT_API_KEY =", MATERIALS_PROJECT_API_KEY))
+            elif 'ORGANIZATION_ID = ' in line:
```

Comment:
The implementation can be more compact and generic for any variable name from kwargs without specifically spelling `ACCOUNT_ID` etc.